File: LojaVendeTudo.API/Models/Produto.cs
using LojaVendeTudo.API.Models;
using DatabaseManager.Validadadores;
using DatabaseManager.Acessos;

namespace LojaVendeTudo.API.Models
{
    public class Produto : AcessoBanco
    {
        [IdSequence]
        public int ProdutoID { get; set; }
        public string Descricao  { get; set; }
        public string Nome { get; set; }
	    public string Marca { get; set; }
	    public string Fabricante { get; set; }
	    public string Peso { get; set; }
	    public decimal PrecoUnitario { get; set; }
        public string ImgSrc { get; set; }
        public int fk_Fornecedor { get; set; }
        public int fk_Categoria { get; set; }
    }
}

File: Angular/src/app/Modelos/produto.ts
export class Produto{
      ProdutoID: number = 0;
      Descricao: string = '';  
      Nome: string = ''; 
      Marca: string = ''; 
      Fabricante: string = ''; 
      Peso: string = ''; 
      PrecoUnitario: number = 0; 
      ImgSrc: string = '';
      fk_Fornecedor: number = 0; 
      fk_Categoria: number = 0; 
      qtd: number = 0;
}